src/accel/TrackingAction.cc

```diff
@@ -29,9 +35,10 @@ void TrackingAction::PreUserTrackingAction(const G4Track* track)
 {
     // Check against list of supported celeritas particles
     PDGNumber pdg{track->GetDefinition()->GetPDGEncoding()};
-    if (true)
+    if (params_->params->particle()->find(pdg))
```

Comment:
We might as well save just a pointer to the particle params? Save a few indirections per track and make it more clear what data this class actually uses.

Reply:
Never mind, I forgot that the params->params might not be initialized when the constructor is called.

Maybe add a
```c++
CELER_EXPECT(params_->params);
CELER_EXPECT(*transport_);
```
to the beginning of the function?